### ISSUE TITLE
OpenVPN DNS / Internet issue / NAT issue??

### ISSUE THREAD

**Issue by benspring** [Nyr/openvpn-install#528]
I have the OpenVPN server hosted on an AWS instance. When I connect to it, I can ping other machines in the subnet which is all good. But I cannot reach the internet. I can ping 8.8.8.8 but not google.com.

Routing table (top route is before running OpenVPn and the second one is whilst running OpenVPN)
![fsbrszk](https://user-images.githubusercontent.com/23476599/46634113-c7342500-cb47-11e8-93fa-4491ee199deb.png)

The private IP is correct and so is the public IP. I initially set it up to use Googles DNS but since changed to cloudflare, then tried remove it all but to no avail. 

Question is: Am I doing something wrong? **Can I make this easier and just let the user use their own DNS**. The machines they reach using the OpenVPN are just private machines (only accessible on with a internal IP address). Accessing anything could be routed via them. 

I set up the following NAT on the instance running the OpenVPN:

![screenshot from 2018-10-08 22-30-15](https://user-images.githubusercontent.com/23476599/46634746-c8fee800-cb49-11e8-828d-2a886489087e.png)

After creating the NAT gateway, any ideas on what to do next? The program recognises I am behind a NAT and asks the for public IP address. I just give it the elastic IP address that I generated on Amazons AWS. Should I give it the NAT gateway or something? I am very confused.

Server:

> proto udp
> dev tun
> sndbuf 0
> rcvbuf 0
> ca ca.crt
> cert server.crt
> key server.key
> dh dh.pem
> auth SHA512
> tls-auth ta.key 0
> topology subnet
> server 10.8.0.0 255.255.255.0
> ifconfig-pool-persist ipp.txt
> push "redirect-gateway def1 bypass-dhcp"
> push "dhcp-option DNS 1.1.1.1"
> keepalive 10 120
> cipher AES-256-CBC
> comp-lzo
> user nobody
> group nogroup
> persist-key
> persist-tun
> status openvpn-status.log
> log-append openvpn.log
> verb 5
> crl-verify crl.pem
> script-security 2
> up /etc/openvpn/update-resolv-conf
> down /etc/openvpn/update-resolv-conf

Client:

> client
> dev tun
> proto udp
> sndbuf 0
> rcvbuf 0
> remote 18.202.129.195 1194
> resolv-retry infinite
> nobind
> persist-key
> persist-tun
> remote-cert-tls server
> auth SHA512
> cipher AES-256-CBC
> comp-lzo
> setenv opt block-outside-dns
> key-direction 1
> verb 6

**Comment by benspring** [comment]
I want this removed:` /sbin/ip route add 0.0.0.0/1 via 10.8.0.1...`

Just want to leave this line: `/sbin/ip route add 10.0.0.20/24 via 10.8.0.1`

**Comment by benspring** [comment]
TL;DR: I just want to direct 10.0.0.* traffic through the VPN and nothing else. Any ideas?? This will stop the NAT problem too.

**Comment by benspring** [comment]
https://forums.openvpn.net/viewtopic.php?t=17942 

For anyone wanting to do the same ^^
